### ISSUE TITLE
Fine tune the existing Model 

### ISSUE THREAD

**Issue by muhammadabdullah34907** [NathanUA/U-2-Net#42]
Hi @NathanUA 

I want to fine tune the existing model(u2net). I understand that we can resume the training simply as discussed in this issue.
https://github.com/NathanUA/U-2-Net/issues/33

```
if(model_name=='u2net'):
    net = U2NET(3, 1)
elif(model_name=='u2netp'):
    net = U2NETP(3,1)
net.load_state_dict(torch.load(saved_model_dir))

if torch.cuda.is_available():
    net.cuda()

```

In addition, to resume the training from where exactly it was, one usually needs to save and load the optimizer (especially for Adam)

So my question is In addition to weights, how can I load the optimizer state as well ?

**Comment by bluesky314** [comment]
I dont think opt state dict is given in pth file so you will have to restart it. Doing learning rate warmup will be a good idea.
